client/views/msg_preview_toggle.tpl

```diff
@@ -0,0 +1,10 @@
+{{#preview}}
+<button class="toggle-button {{#if shown}} opened{{/if}}"
+  data-url="{{link}}"
+  {{#equal type "image"}}
+    aria-label="Toggle image preview"
```

Comment:
No, it was always `aria-label`, not tooltips 😅
I'm up for making these tooltips, but mind doing that later after merging? I'm going to be fairly busy this weekend, hence me addressing comments at 5am :)